packages/hash/task-executor/src/index.ts

```diff
@@ -0,0 +1,35 @@
+import { createServer, IncomingMessage, ServerResponse } from "http";
+import { executeTask } from "./execution";
+
+const port = 5010;
+
+const requestHandler = async (
+  request: IncomingMessage,
+  response: ServerResponse,
+) => {
+  const url = request.url;
+  let result = "Fallback response";
+  try {
+    if (url === "/python") {
+      result = await executeTask("python", ["-m", "src.tasks.demo"]);
+    }
+  } catch (err: any) {
+    console.log(err);
+    response.statusCode = 500;
+    response.end(err.toString());
+    return;
+  }
+
+  response.end(result);
+};
+
+// eslint-disable-next-line @typescript-eslint/no-misused-promises -- createServer doesn't accept a `Promise<void>` as it expects just a void return signature but we don't want to block
+const server = createServer(requestHandler);
```

Comment:
Small thing, but we can get rid of this eslint-disable by using createServer in another way. The handler given in this constructor param is equivalent to a `.on('request' ...)` on the `server`.
Using an event listener allows you to pass a promise, and wouldn't be considered a misused promsie.

see https://nodejs.org/api/http.html#httpcreateserveroptions-requestlistener

Reply:
What the hek, Node. I guess Alex or Ben might know what to do in this case

Reply:
I hope it's fair to assume that the handlers won't block by default

Reply:
Yeah Alex and I had a discussion about this yesterday. My thought is we shouldn't care too much right now. If it gets to the point where we worry about blocking requests, scalability, etc. we delete this package and swap to Temporal or Argo or whatever :)